### ISSUE TITLE
PanelMenu: attribute id is set twice for top-level entry that is not a router link

### ISSUE THREAD

**Issue by jBuchholz590** [primefaces/primeng#10144]
**I'm submitting a ...**
```
[*] bug report
```

**Current behavior**
In the PanelMenu the top layer is rendered inside PanelMenu component.
Depending on the existence of a routerLink for the underlying MenuModel item one of two `<a>` elements is rendered.
If a routerLink is set the `<a>` is given an id correctly as specified via the MenuModel (`[attr.id]="item.id"`).

If on the other hand no routerLink is present, the id attribute is assigned twice to the `<a>` element.
First (as expected) `[attr.id]="item.id"` is called, then `"[attr.id]=item.id + '_header'"` resulting in the id having the '_header' suffix.

**Expected behavior**
The id is set only once to the id provided by the MenuModel item.

If the ids were intended to be different for the `<a>` elements the difference should be more self-explanatory (e.g. `withRouterLink` and `withoutRouterLink`)

* **PrimeNG version:** 11.3.3

**Comment by mertsincan** [comment]
Hi,

So sorry for the delayed response! Improvements have been made to many components recently, both in terms of performance and enhancement. Therefore, this improvement may have been developed in another issue ticket without realizing it. You can check this in the documentation. If there is no improvement on this, can you reopen the issue so we can include it in our roadmap? 
Please don't forget to add your feedback as a comment after reopening the issue. These will be taken into account by us and will contribute to the development of this feature. Thanks a lot for your understanding!

Best Regards,
